src/averell/utils.py

```diff
@@ -284,3 +284,40 @@ def read_features(corpus_folder):
 
     features = sorted(features, key=lambda i: i['poem_title'])
     return features
+
+
+def pretty_string(text, n_words):
+    """Add a line break every number of words into a text to create multiline
+    cells to use in `tabulate_corpora`
+
+    :param text: String to be converted
+    :param n_words: Number of words in which the line break will be added
+    :return: String with line break every number of words entered
+    :rtype: str
+    """
+    words = text.split()
+    grouped_words = [' '.join(words[i: i + n_words]) for i in
+                     range(0, len(words), n_words)]
+    return '\n'.join(grouped_words)
+
+
+def get_main_corpora_info():
+    """Create dict with the main corpora info saved in CORPORA_SOURCES
+
+    :return: Dictionary with the corpora info to show
+    :rtype: dict
+    """
+    table = []
+    for corpus_info in CORPORA_SOURCES:
+        corpus_id = CORPORA_SOURCES.index(corpus_info) + 1
+        table.append({
+            "id": corpus_id,
+            "name": pretty_string(corpus_info["name"], 2),
+            "size": corpus_info["properties"]["size"],
```

Comment:
If you don't want lines that long maybe you can add `corpus_info["properties"]` as a variable and then just:
`corpus_props = corpus_info["properties"]`
`"size": corpus_props["size"],`

Reply:
Done